Tests/SentryTests/Integrations/Feedback/SentryFeedbackTests.swift

```diff
@@ -190,6 +190,130 @@ class SentryFeedbackTests: XCTestCase {
 
         }
     }
+    
+    func testFeedbackNotSubjectToSampling() throws {
+        let options = Options()
+        options.dsn = TestConstants.dsnAsString(username: "SentryFeedbackTests")
+        options.sampleRate = 0.0 // Sample rate that would normally filter out all events
+
+        let transport = TestTransport()
+        let transportAdapter = TestTransportAdapter(transports: [transport], options: options)
+
+        let client = SentryClient(
+            options: options,
+            transportAdapter: transportAdapter,
+            fileManager: try XCTUnwrap(SentryFileManager(options: options, dispatchQueueWrapper: TestSentryDispatchQueueWrapper())),
+            deleteOldEnvelopeItems: false,
+            threadInspector: TestThreadInspector.instance,
+            debugImageProvider: TestDebugImageProvider(),
+            random: TestRandom(value: 1.0),
+            locale: Locale(identifier: "en_US"),
+            timezone: try XCTUnwrap(TimeZone(identifier: "Europe/Vienna"))
+        )
+        let hub = TestHub(client: client, andScope: nil)
+
+        SentrySDKInternal.setCurrentHub(hub)
+        
+        let feedback = SentryFeedback(
+            message: "Test feedback message",
+            name: "Test User",
+            email: "test@example.com",
+            source: .widget
+        )
+
+        SentrySDK.capture(feedback: feedback)
+        
+        // Verify that the feedback was captured and sent despite the 0.0 sample rate
+        let lastSentEventArguments = try XCTUnwrap(transportAdapter.sendEventWithTraceStateInvocations.last)
+        let capturedFeedback = try XCTUnwrap(lastSentEventArguments.event)
+        print(capturedFeedback)
+//
+//        XCTAssertEqual(capturedFeedback.message, "Test feedback message")
+//        XCTAssertEqual(capturedFeedback.name, "Test User")
+//        XCTAssertEqual(capturedFeedback.email, "test@example.com")
```

Comment:
This asserts should be enabled, right?

Reply:
whoops, no, they should be removed. there's currently no way to inspect the event for these particular attributes, since SentryFeedback (like SentryUserFeedback before it) don't actually inherit from SentryEvent.

Reply:
Good catch, I can't check these specific attributes since SentryFeedback (like SentryUserFeedback before it) doesn't inherit from SentryEvent, but, I will see if I can check something in the event metadata that indicates it's actually a user feedback.

Reply:
added assertions on error types in https://github.com/getsentry/sentry-cocoa/pull/5692/commits/b4e4c51c78f7872284b782fdfc26434fdcd4b42c